### ISSUE TITLE
Your minions with "can't attack" don't fight when summoned by Duel!

### ISSUE THREAD

**Issue by troggnostupidhs** [HearthSim/hs-bugs#1157]
https://us.forums.blizzard.com/en/hearthstone/t/duel-pulling-rag-interaction/3568

> 
> Hi I recently started playing “big Paladin” in wild, and there’s a interesting interaction with Duel and Rag. If you play Duel and pull your Rag it won’t “fight” the opponents pulled minion. But if you play Duel and pull you’re opponents rag it will fight, I expect this to have to do with the code? You pull x and y minions from both deck then your target minion attacks opponents minion, but rags code overrights that with the “can’t attack” keyword.
> 
> (Sorry that I don’t have any source of this I almost strictly play on mobile.)

Rag
AAEBAaToAgL2AsD9Ag76AdwD9AWvB90K6w+dFbuvArjHAu3SAoT8AuOGA+yGA4acAwA=

Can't Attack
AAEBAaToAgAP+gHcA90E9AWvB90K6w+dFbjHAsXHAu3SAoT8AuOGA+yGA4acAwA=

**Comment by czhihong** [comment]
I don't think this is a bug. The way Peter Whalen explains it, the intended behaviour is that "Your minion attacks theirs". If we assume that this implicitly means "if they are able to do so", the current behaviour with own/opposing Ragnaros makes sense. I can drop them a message to clarify.

https://twitter.com/LegendaryFerret/status/1111641807333412866

**Comment by troggnostupidhs** [comment]
Mass Hysteria and Supercollider cause them to attack. And allegedly from the report, they will attack if they are summoned for your opponent. It seems like an inconsistency that they do not fight when they are summoned for the player. 

AAEBAaCsAwAP+wGQApwC3QTlBLkG0wqPDskT8LsC0cECxccCpvACl4cDrpsDAA==

**Comment by Jetz72** [comment]
Acknowledged by Blizzard:
https://us.forums.blizzard.com/en/hearthstone/t/known-issues-list-2019-updated-may-22nd/64/8
>Duel! will not force minions to fight if your minion contains “Can’t Attack”
